README.md

```diff
@@ -1 +1,101 @@
-# Hello_Rails_React
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello rails react <a name="about-project"></a>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>React on Rails</summary>
+  <ul>
+    <li><a href="https://ruby-doc.org/core-3.1.2/">React</a></li>
+  </ul>
+    <ul>
+    <li><a href="https://ruby-doc.org/core-3.1.2/">Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add new greetings**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+Clone this repository by using the command line :
+
+- `git clone https://github.com/mbdesigns1989/Hello_Rails_React`
+
+### Install
+
+Install this project with:
+
+`bundle install`
+
+### Usage
+
+To run the project, execute the following command:
+
+`rails db:create  rails db:seed yarn build --watch rails s `
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+  👥 **Mohamed Badr**
+
+- GitHub: [@mbdesigns1989](https://github.com/mbdesigns1989)
+- LinkedIn: [mohamed badr](https://www.linkedin.com/in/mohamed-badr-mb/)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add more greetings**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page].
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project by staring it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT]
```

Comment:
- [ ] Kindly include a valid LICENSE link to the LICENSE section in your README file. Adding an open-source license to your repository ensures that others can use, copy, modify, and contribute back to your project. It's an important step when creating an open-source project. 😎